sqlmesh/core/snapshot/definition.py

```diff
@@ -942,7 +942,7 @@ def _model_data_hash(model: Model) -> str:
             data.append(macro_name)
             data.append(macro.definition)
 
-        for package in model.jinja_macros.packages.values():
+        for _, package in sorted(model.jinja_macros.packages.items(), key=lambda x: x[0]):
```

Comment:
Nit: The key argument isn't really needed here.

Reply:
what do you mean?

Reply:
we need it for sorting

Reply:
I don't think it's a big deal. I meant `sorted` will use the first element of x automatically for sorting.

Reply:
big brain

```python
>>> sorted({"b": 1, "a": 2}.items())
[('a', 2), ('b', 1)]
```